1-getting-started/Translations/README.fr.md

```diff
@@ -0,0 +1,15 @@
+# Guide de démarage IoT
+Dans cette section du tutoriel, vous serez initié à l'Internet des objets et apprendrez les concepts de base, notamment la construction de votre premier projet IoT « Hello, World!» connecté à l'infrastrucutre infonuagique. Ce projet est une veilleuse qui s'allume quand la lumière mesurée par un capteur diminue.
+
+![La DEL connecté au WIO s'allume et s'éteint en fonction de la luminosité](../../images/wio-running-assignment-1-1.gif)
+
+## Sujets
+
+1. [Introduction au IoT](../lessons/1-introduction-to-iot/translations/README.fr.md)
+1. [Approfondir les IoT](../lessons/2-deeper-dive/translations/README.fr.md)
+1. [Intéragir avec le monde physique à l'aide de capteur et actionneur](../lessons/3-sensors-and-actuators/translations/README.fr.md)
+1. [Connecter votre appareil à Internet](../lessons/4-connect-internet/translations/README.fr.md)
+
+## Crédits
+
+Tous les leçons ont été écrite ♥️ par [Jim Bennett](https://GitHub.com/JimBobBennett)
```

Comment:
Let’s get a linguistic review here please - there are gender and verb agreement errors here: 
toutes les leçons ont été écrites par…